resources/codegen_inputs/test_records.rs

```diff
@@ -7,10 +7,11 @@
 #![parse_module(read_fonts::codegen_test::records)]
 
 table BasicTable {
+    padded_offset: Offset32<PadLikeCmap>,
```

Comment:
can we remove these changes now? It looks like they were added to explore what wasn't working, but I think this now just duplicates stuff that exists in `test_offsets_arrays.rs`.

Reply:
This is not adequately tested by test_offsets_arrays. I could make it so I suppose.

Reply:
oh I see, I didn't understand what made this special when I first looked.